### ISSUE TITLE
[1.12.2] Hypoinfusers won't fill in SMP

### ISSUE THREAD

**Issue by zcorcoran** [CoFH/Feedback#1062]
This is a weird one.

On my server, hypoinfusers refuse to fill by any means; I've tried the Fluid Transposer, the IE bottling machine, and crafting the hypoinfuser with a potion bottle. I've also created potion fluid with both the IE mixer and a chain of Alchemical Imbuers just in case it was an issue with the fluid itself, but the hypoinfuser refused both. This happens both when the hypoinfuser is and isn't enchanted.

However, in singleplayer, it just works. I thought it might be a mod interaction, so I tested it offline and it worked before I removed anything. Since it didn't seem to be any mod that was present causing the problem, I added every mod that's server side only to my clientside modpack and... it still works. I also synchronized versions of Forge and the entire config directory between the client and server, still nothing.

I'm at a loss. In addition to mods being 100% synced between client and server and the disparity continuing, all mods are loading in the same order between client and server, or should be, because the filenames are exactly the same between the two.

Both client and server have:
Thermal Innovation 1.12.2 0.2.1.6
CoFH Core 1.12.2 4.4.1.12
Forge 14.23.3.2669

**Comment by KingLemming** [comment]
I am unable to replicate this. Tested on a server and it works fine.

**Comment by zcorcoran** [comment]
I thought that might happen, yeah. It's bizarre. I'll keep messing around with it and try to find differences between the two setups.

**Comment by KingLemming** [comment]
Try a new world on the server. Just for testing, obviously.

**Comment by KingLemming** [comment]
Also, are you able to fill the Quiver?

**Comment by zcorcoran** [comment]
Nope, the quiver does the same thing. It fills in singleplayer but not on the server.
It might be worth noting that the fluid transposer _accepts_ the items, in that it goes from the input slot to the middle slot, but on the server it just sits there after that.

**Comment by KingLemming** [comment]
Try removing IE. TF refreshes its fluids but I'm wondering if there's an edge case now.

**Comment by Drullkus** [comment]
Oh, this should be an interesting bit. KL, do the Thermal Potion fluid work with IE’s Potion fluid? Like, is the NBT compatible?

**Comment by KingLemming** [comment]
It is actually, yes. I coordinated with Blu on that.

**Comment by Drullkus** [comment]
Hah, awesome!
